### PR TITLE
Remove perms

### PR DESCRIPTION
**Pull request by mikecirioli**
See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).
Initial work to remove (`deprecate`?) dangerous permissions and consolidate them under `Jenkins.ADMINISTER`.  A new permission type, `Jenkins.CONFIGURE_JENKINS` is introduced to allow non-admin configuration of non-dangerous options.

### Proposed changelog entries

* Entry 1: Issue, Human-readable Text
* ...

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### CODE REVIEW

File: core/src/main/resources/hudson/model/Messages.properties [hunk 1]
```diff
@@ -163,6 +163,7 @@ Hudson.AdministerPermission.Description=\
   This permission grants the ability to make system-wide configuration changes, \
   as well as perform highly sensitive operations that amounts to full local system access \
   (within the scope granted by the underlying OS.)
+Hudson.ConfigureJenkinsPermission.Description=Configure bits of jenkins
```
Comment (aHenryJard) [inline core/src/main/resources/hudson/model/Messages.properties:166]: Can we use Jenkins instead of Hudson ?

Comment (aHenryJard) [reply core/src/main/resources/hudson/model/Messages.properties:166]: "Configure bits of jenkins" => "Configure Jenkins except for secure risked configuration" ?

Comment (mikecirioli) [reply core/src/main/resources/hudson/model/Messages.properties:166]: Good question, i don't rightly know.

Comment (mikecirioli) [reply core/src/main/resources/hudson/model/Messages.properties:166]: That is just placeholder text, i also didn't touch any of the localized properties files

**Pull request closed without merging**